test/index.test.ts

```diff
@@ -212,6 +212,20 @@ describe('Test post categories', () => {
     expect(res.body.message).toBe('تم إضافة الخدمة بنجاح');
   });
 });
+describe('Test post categories', () => {
+  it('should return status 201', async () => {
+    const res = await supertest(app)
+      .post('/api/v1/categories')
+      .send({
+        name: 'mostafa',
+        description: '',
+        image: 'dsfdsfsdggs',
+      })
+      .expect(400)
+      .expect('Content-Type', /json/);
+    expect(res.body.status).toBe(400);
```

Comment:
You already tested the status code in line `224`, try to test the response message